src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,30 +1,61 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+export enum STATUSES {
+  COMPLETED = 'completed',
+  ACTIVE = 'active',
+  ALL = 'all',
+}
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+interface FilterProps {
+  setStatus: (value: string) => void;
```

Comment:
The `setStatus` function expects a string, but it would be more type-safe to use the `STATUSES` enum. Consider changing the type of `setStatus` to `(value: STATUSES) => void` to ensure that only valid status values are passed.

Reply:
> The `setStatus` function expects a string, but it would be more type-safe to use the `STATUSES` enum. Consider changing the type of `setStatus` to `(value: STATUSES) => void` to ensure that only valid status values are passed.

Good point